### PR TITLE
Improve RTL version of login page

### PR DESCRIPTION
**Pull request by amirmasoud**

### CODE REVIEW

**Comment by emptynick** [comment]
I just changed your base-branch from `1.1` to `1.x` because the css file needs publishing. (And I by mistake merged 1.1 into your master, sorry for that)

By the way, cant you just [flip the image](https://www.w3schools.com/howto/howto_css_flip_image.asp)?

**Comment by amirmasoud** [comment]
@emptynick I have tried it, it cause whole body to flip, background-color is set on body tag, for this solution I needed more changes to be done, I just did the easiest way.

File: resources/views/login.blade.php [hunk 1]
```diff
@@ -14,7 +14,11 @@
     @endif
     <style>
         body {
+            @if (config('voyager.multilingual.rtl'))
```
Comment (emptynick) [inline resources/views/login.blade.php:17]: This config doesn't exist in 1.x anymore.
Please refer to this line:
https://github.com/the-control-group/voyager/blob/bb9646169ac30e2b230495c82bab42d89cb4f717/resources/views/master.blade.php#L2

**Review by emptynick — CHANGES_REQUESTED** [review]

File: resources/views/login.blade.php (hunk 1 above)
Comment (emptynick) [inline resources/views/login.blade.php:17]: Thats how we decide wether to use RTL or not.
```suggestion
            @if (__('voyager::generic.is_rtl') == 'true')
```

**Review by emptynick — CHANGES_REQUESTED** [review]

**Review by emptynick — CHANGES_REQUESTED** [review]
I'm not really happy with having another image just so its mirrored.
In my opinion this can be done through CSS.

**Comment by emptynick** [comment]
Closing as the fork was deleted.

**Pull request closed without merging**